values.yaml

```diff
@@ -18,6 +18,24 @@ agent:
     requests:
       cpu: 600m
       memory: 968Mi
+  nodeSelector:
+    beta.kubernetes.io/instance-type: t3.medium
```

Comment:
- you did not update `agent-deployment.yaml` for this 
- I would use `nodeSelector: {}` for the default value, so that if we use any random type of node, we don't constrain the user to override the nodeSelector for it to work in his cluster.

Reply:
👍